src/reports/html.test.js

```diff
@@ -41,6 +46,46 @@ window.RESTQA_RESULT = {
   "foo": "bar"
 }`
     expect(fs.readFileSync(fileResult).toString('utf-8')).toEqual(expectedFileResult.trim())
+    expect(fs.readFileSync(fileResult).toString('utf-8')).toEqual(expectedFileResult.trim())
+    expect(mockOpen).toHaveBeenCalled()
+    expect(mockOpen.mock.calls[0][0]).toEqual(`file://${FOLDER}/index.html`)
```

Comment:
This line should be enough.
```suggestion
    expect(mockOpen).toHaveBeenCalledWith(`file://${FOLDER}/index.html`)
```